source/navigation.js

```diff
@@ -35,6 +35,7 @@ import SettingsView from './views/settings'
 import CreditsView from './views/settings/credits'
 import PrivacyView from './views/settings/privacy'
 import LegalView from './views/settings/legal'
+import KSTOSchedule from './views/streaming/radio/schedule'
```

Comment:
This should be reorganized so that it's exported from the same file (views/streaming) as the radio view itself. And ideally all three tabs and associated views would be imported from views/streaming.